api/v1alpha1/server_types.go

```diff
@@ -42,6 +42,32 @@ const (
 	ServerPoweringOffPowerState ServerPowerState = "PoweringOff"
 )
 
+// Capacity is a disk size in Bytes.
+type Capacity int64
+
+// Capacity multipliers.
+const (
+	Byte     Capacity = 1
+	KibiByte          = Byte * 1024
+	KiloByte          = Byte * 1000
+	MebiByte          = KibiByte * 1024
+	MegaByte          = KiloByte * 1000
+	GibiByte          = MebiByte * 1024
+	GigaByte          = MegaByte * 1000
+	TebiByte          = GibiByte * 1024
+	TeraByte          = GigaByte * 1000
+)
```

Comment:
What about using `resource.Quantity` from "k8s.io/apimachinery/pkg/api/resource"?

Reply:
This

Reply:
What format do we prefer? DecimalSI or BinarySI?

Reply:
If the `Quantity` represents floating-point (like `CPU`) `DecimalSI` is being used. In case of `Storage`, `Memory` and so on you can safely use `BinarySI`.

Reply:
redfish reports capacity in bytes.

so from our mockup redfish 8000000000000 bytes turn into `7812500000Ki`. Was hoping it would automatically turn into Ti.

Reply:
The exponent will be picked by the library in order to avoid floating numbers (which could lead to precision errors). 
If you like to println/log in a human readable way, you can still enforce the scale `ScaledValue(resource.Giga)`.

Reply:
I tried that, but it does not affect the outcome of Marshalling. Anyway, the type is now `resource.Quantity`, so other services can easily parse it.